salt/states/win_network.py

```diff
@@ -277,21 +282,16 @@ def managed(name,
                     ret['comment'] = ('Failed to disable interface \'{0}\''
                                       .format(name))
         else:
-            ret['comment'] += ' (already disabled)'
+            ret['comment'] += '(already disabled)'
```

Comment:
I am curious why you removed the leading space. Doesn't this get added onto:
```
'Interface \'{0}\' is up to date.'.format(name)
```
and hence having that space helps readability?

Reply:
My bad. Adding it back.